galgebra/printer.py

```diff
@@ -1,5 +1,80 @@
 r"""
 ANSI Enhanced Text Printing, Text Printer and LaTeX Printer for all Geometric Algebra classes
+
+:math:`\LaTeX` printing
+-----------------------
+
+.. note::
+
+    :mod:`galgebra` works out of the box with the usual
+    :ref:`sympy printing <sympy:tutorial-printing>` and will show Latex in
+    IPython by default. In many cases, all that is needed is::
+
+        sympy.init_printing(
+            use_latex='mathjax',
+            latex_printer=galgebra.printer.latex,
+            # described below
+            omit_function_args=True,
+            omit_partial_derivative_fraction=True,
+        )
+
+    The rest of this section primarily describes an orthogonal feature for
+    writing out ``.tex`` files with :func:`print`.
+
+The latex printer is turned on with the :func:`Format` function
+
+.. function:: Format(Fmode=True, Dmode=True, ipy=False)
+
+where Fmode is the function printing mode that surpresses printing arguments,
+Dmode is the derivative printing mode that does not use fractions, and
+ipy=True is the Ipython notebook mode that does not redirect the print output.
+
+The latex output is post processed and displayed with the function
+
+.. function:: xpdf(filename='tmplatex.tex', debug=False)
+
+where filename is the name of the tex file one would keep for future
+inclusion in documents and debug=True would display the tex file
+immediately.
+
+There are three options for printing multivectors in latex.  They are
+acessed with the multivector member function
+
+.. function:: galgebra.mv.Mv.Fmt(self, fmt=1, title=None)
+
+where fmt=1, 2, or 3 determines whether the entire multivector A is
+printed entirely on one line, or one grade is printed per line, or
+one base is printed per line.  If title is not None then the latex
+string generated is of the form::
+
+    title + ' = ' + str(A)
+
+where it is assumed that title is a latex math mode string. If title
+contains '%' it is treated as a pure latex math mode string.  If it
+does not contain '%' then the following character mappings are applied::
+
+    'grad' -> '\bm{\nabla} '
+    '*'    -> ''
+    '^'    -> '\W '
+    '|'    -> '\cdot '
+    '>'    -> '\lfloor '
+    '<'    -> '\rfloor '
+
+In the case of a print statement of the form::
+
+    print(title, A)
+
+everthing in the title processing still applies except that the multivector
+formatting is one multivector per line.
+
+For print statements of the form::
+
+    print(title)
+
+where no program variables are printed if title contains `#` then title
+is printed as regular latex line.  If title does not contain `#` then
+title is printed in equation mode. `%` has the same effect in title as
```

Comment:
Should we merge the `#` `%` doc and code in one place later?

Reply:
Yeah, probably